boards/acd52832/src/main.rs

```diff
@@ -119,84 +118,49 @@ pub unsafe fn reset_handler() {
 
     // GPIOs
     let gpio_pins = static_init!(
-        [&'static dyn kernel::hil::gpio::InterruptValuePin; 14],
+        [&'static dyn kernel::hil::gpio::InterruptValuePin; 7],
```

Comment:
@bradjc is this refactoring expected?

Reply:
Yes...the ACD pins seems to be all wrong, I probably copied them and forgot to update.